documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/reindexing/derivedfrom/Book.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.documentation.mapper.orm.reindexing.derivedfrom;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexingDependency;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.ObjectPath;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.PropertyValue;
+
+// Note: this example is too complex for its own good, but I can't think of a much simpler one.
+//tag::include[]
+@Entity
+@Indexed
+public class Book {
+
+	@Id
+	@GeneratedValue
+	private Integer id;
+
+	@FullTextField(analyzer = "name")
+	private String title;
+
+	@ElementCollection
+	private List<String> authors = new ArrayList<>(); // <1>
+
+	public Book() {
+	}
+	//tag::getters-setters[]
+
+	public Integer getId() {
+		return id;
+	}
+
+	public void setId(Integer id) {
+		this.id = id;
+	}
+
+	public String getTitle() {
+		return title;
+	}
+
+	public void setTitle(String title) {
+		this.title = title;
+	}
+
+	public List<String> getAuthors() {
+		return authors;
+	}
+
+	public void setAuthors(List<String> authors) {
+		this.authors = authors;
+	}
+	//end::getters-setters[]
+
+	@Transient // <2>
```

Comment:
It's not strictly necessary because the access type in this entity is `field`, indeed. But it doesn't hurt, and personally if this were my codebase I would use this annotation anyway, just to document the fact this is a transient property.

What's more important is that if anyone copy/pastes this example, it will work regardless of the access type in their entity. Which is great, since I don't want to explain how to map a Hibernate ORM entity.